test/reader_test/fio_reader_test.cpp

```diff
@@ -21,7 +21,7 @@
 //      Author: Eugen Wintersberger <eugen.wintersberger@desy.de>
 //
 #include <boost/test/unit_test.hpp>
-#include <boost/test/floating_point_comparison.hpp>
+#include <boost/test/tools/floating_point_comparison.hpp>
```

Comment:
with this change, i think it's better to change the boost requirements in installation: Boost (>=1.55). In old boost, this header file is in another path which previously was used.

Reply:
Done